app/src/main/java/com/ayush/weatherapp/entities/DailyDataEntity.java

```diff
@@ -1,85 +1,107 @@
-package com.ayush.weatherapp.retrofit.weatherApi.pojo;
+package com.ayush.weatherapp.entities;
 
 import android.os.Parcel;
 import android.os.Parcelable;
-import com.google.gson.annotations.SerializedName;
-
-public class DailyData implements Parcelable {
-
-  @SerializedName("time") private long time;
-  @SerializedName("summary") private String summary;
-  @SerializedName("icon") private String icon;
-  @SerializedName("sunriseTime") private int sunriseTime;
-  @SerializedName("sunsetTime") private int sunsetTime;
-  @SerializedName("temperatureHigh") private double temperatureHigh;
-  @SerializedName("temperatureLow") private double temperatureLow;
-  @SerializedName("apparentTemperatureHigh") private double apparentTemperatureHigh;
-  @SerializedName("apparentTemperatureLow") private double apparentTemperatureLow;
-  @SerializedName("windSpeed") private double windSpeed;
-
-  DailyData(Parcel in) {
+
+public class DailyDataEntity implements Parcelable {
+  public static final Creator<DailyDataEntity> CREATOR = new Creator<DailyDataEntity>() {
```

Comment:
below after constructor is the preferred location

Reply:
The autoformat reformats the code this way. Probably because of the static final.